js/counties.js

```diff
@@ -36,7 +36,7 @@ function displayCountyData(error, activeCountyData) {
     // extract the topojson to geojson
     var geojson = topojson.feature(activeCountyData, activeCountyData.objects.state);
     var yeargeo = geojson.features.filter(function(d) {
```

Comment:
question I put on your previous PR still stands - should we change the way the county data is processed so that it only includes 2015? Currently preprocessing does it for multiple years, right?

Reply:
oh! yes. i will do that.